evm_loader/program/src/evm/memory.rs

```diff
@@ -55,6 +69,8 @@ impl Memory {
                 data,
                 capacity,
                 size: v.len(),
+                #[cfg(feature = "tracing")]
+                tracer: None,
```

Comment:
It is better to panic here or cause a compiler error. This method should never be called with the feature = 'tracing'.
The same for other similar methods

Reply:
My complain that if this method is used with tracer you will receive incorrect data without any indication of it. There need to be clean error.

Reply:
The line you mentioned above is used inside the `from_buffer` method. I do not understand exactly what you are suggesting.

Reply:
Fine. If you're absolutely sure you won't use EVM serialization with a tracer ever.

Reply:
Unless someone removes `#[serde(skip)]` attribute, it should be safe.